packages/sfui/frameworks/vue/components/SfSelect/SfSelect.vue

```diff
@@ -62,8 +60,8 @@ const modelProxy = computed({
     data-testid="select"
   >
     <select
+      v-model="internalValue"
```

Comment:
this wont work for sure when you will change model-value from outside, meaning 
1. when select is changed emit should be done
2. when model is changed outside should reflect in select

Select component as any framework UI components should not keep its state, that why you should have model value outside your component and component itself is just UI representation.

Reply:
> Select component as any framework UI components should not keep its state, that why you should have model value outside your component and component itself is just UI representation.

I thoroughly agree. Unfortunately with Vue it does not work as expected (see the documentation example above) **unless** we use the v-model on that component in the parent component.

Therefore, if we agree that this component should not have any internal state (which I am fine with), we should mark `modelValue` as `required` to avoid such bugs. And this will be a breaking change.

Reply:
I would suggest using proxy with internal as it is done in `SfInput` 

```
const internalState = ref<string | number>();
const selectModel= computed({
  get: () => modelValue.value ||  internalState.value,
  set: (value) => {
    emit('update:modelValue', value);
    internalState.value = value;
  },
});
```